packages/spectacle/src/components/slide-layout.tsx

```diff
@@ -66,6 +66,7 @@ const List = ({
           {title}
         </Heading>
       ) : null}
+      {/* @ts-ignore TODO: Resolve this in follow-up */}
       <List {...listProps}>
```

Comment:
TS yelling about spreading `listProps` here, need to just do a little bit of TS shenanigans.